packages/server/validators/events.js

```diff
@@ -193,8 +194,41 @@ const eventIdValidator = [
     }),
 ];
 
+const attendeeStatusValidator = [
+  // validate status
+  body("status", "invalid status")
+    .trim()
+    .blacklist("<>")
+    .exists({ checkFalsy: true })
+    .withMessage("status cannot be null or empty")
+    .custom(async (status) => {
+      // Validate status
+      const allowedStatusValues = ["NotInterested", "Interested", "Attending"];
+      if (!allowedStatusValues.includes(status)) {
+        throw new Error(
+          "Invalid status value. Allowed values are: NotInterested, Interested, Attending"
+        );
+      }
+    }),
+  // validate userId (only difference between this and the validator in users is that it validates the body not params)
+  body("userId")
+    .notEmpty()
+    .withMessage("User ID cannot be empty")
+    .blacklist("<>")
+    .isUUID()
+    .withMessage("Invalid user ID provided")
+    .custom(async (value) => {
+      try {
+        await UserController.getUser(value);
+      } catch (error) {
+        throw new Error("User with ID ${value} not found");
+      }
+    }),
```

Comment:
Move this out into a separate validator like `userIdBodyValidator` in `validators/users.js` instead since it keeps the code modular, and it will be easier to reuse for future features. Justin's task is also using a userId body validator so it'll be easier to resolve the merge conflict later.